### DISCUSSION TITLE
Options file to set file-scheme parameters based per Subreddit when downloading saved posts?

### DISCUSSION THREAD

**Discussion by drebob24** [Serene-Arc/bulk-downloader-for-reddit#912] [Q&A]
When saving my Saved posts via : '--user me --saved' can you use the options file to determine how and what is downloaded? 

Essentially I would like to be able to have posts from some subs having one file-scheme and other posts have a different file-scheme. Also the ability for only listed subreddits to be downloaded would be nice.

**Comment by Fakeaccount12312** [comment]
I don't think such complex behaviour is possible or should be a feature of this tool. Instead, I'd advise you to run the command multiple types for each separate file scheme and set of subreddits. You can use --skip-subreddit 'd, e, f' to exclude certain subreddits, or just delete them manually afterwards, which might probably be the easier way unless you do this often. If you want to run it all with one press of a button you can chain commands using &&.

Unfortunately, there is no way to whitelist certain subreddits, you could request that feature in a clearly defined issue. Doing --subreddit only adds this subreddit to the pool of sources, it is not restricting the already given sources. This tool generally only offers commands to blacklist/exclude posts from the given sources, not to whitelist them. It would be a nice thing to be able to do that too, would certainly expand the amount of features this tool has but may be be a bit complicated to implement, that is up to the contributors though.

But the development is a bit stalled atm, so don't expect a quick fix, better just do what I said in the first paragraph and do it manually. The feature to filter given sources to **only include** certain subreddits/media types/urls/extractors/authors/... will probably take some time to be developed, if it is ever considered. Just make do with what you have now.

**Reply by Fakeaccount12312** [reply]
Opened a proper issue now for this topic, not too high of a priority probably but at least now it has been done, and can be referenced in the future. Lets see what comes out of it. The number is #930.
